src/combat.ts

```diff
@@ -51,7 +52,11 @@ function itemOrItemsBallsMacroName(
     return itemOrItems.map(itemOrItemsBallsMacroName).join(", ");
   } else {
     const item = itemOrNameToItem(itemOrItems);
-    return item.name;
+    return !$items`spider web, really sticky spider web, dictionary, NG, Cloaca-Cola, yo-yo, top, ball, kite, yo, red potion, blue potion, adder, red button, pile of sand, mushroom, deluxe mushroom`.includes(
```

Comment:
can you move this list somewhere else and give it a name?

Reply:
A few options for "somewhere else":
 - utils.ts
 - elsewhere in this document
 - The tools file made for this task

Reply:
Placed them higher up in combat.ts